### PR TITLE
migrator: migrating one INSPIRE record

### PR DESCRIPTION
**Pull request by kaplun**
* Introduces "migrator one --recid RECID" to migrate automatically
  a single record directly from INSPIRE legacy.

Signed-off-by: Samuele Kaplun <samuele.kaplun@cern.ch>

### CODE REVIEW

**Comment by kaplun** [comment]
git @jacquerie review --force -s

File: inspirehep/modules/migrator/cli.py [hunk 1]
```diff
@@ -71,6 +74,15 @@ def populate(file_input=None,
 
 
 @migrator.command()
+@click.option('--recid', '-r', type=int, help="recid on INSPIRE")
+def one(recid):
+    print("Migrating http://inspirehep.net/record/{recid}".format(recid=recid))
```
Comment (jacquerie) [inline inspirehep/modules/migrator/cli.py]: This guy should be a `click.echo`.

File: inspirehep/modules/migrator/cli.py [hunk 2]
```diff
@@ -71,6 +74,15 @@ def populate(file_input=None,
...
+    raw_record = requests.get("http://inspirehep.net/record/{recid}/export/xme".format(recid=recid)).content
```
Comment (jacquerie) [inline inspirehep/modules/migrator/cli.py]: This repeats logic from the line above.

File: inspirehep/cli.py [hunk 3]
```diff
@@ -24,9 +24,12 @@
...
 
+import logging
+
 from invenio_base.app import create_cli
 
 from .factory import create_app
 
 
 cli = create_cli(create_app=create_app)
+logging.basicConfig()
```
Comment (jacquerie) [inline inspirehep/cli.py]: These belong to a different commit, with an explanation in the commit message.

File: inspirehep/modules/migrator/cli.py [hunk 4]
```diff
@@ -37,6 +38,8 @@
     add_citation_counts,
     migrate,
     migrate_broken_records,
+    migrate_chunk,
+    split_marc,
```
Comment (jacquerie) [inline inspirehep/modules/migrator/cli.py]: Importing a regexp means you're duplicating the logic that uses it, which then should be abstracted in a helper.
